src/modules/nav/nav.module.ts

```diff
@@ -2,6 +2,8 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule } from '@angular/router';
 
+import { StacheNavComponentService } from './nav.component.service';
```

Comment:
I'm not sold on a nav component service. Brainstorm an idea that doesn't require it. Maybe a component `@Output()`?